### ISSUE TITLE
Gradle Error

### ISSUE THREAD

**Issue by ilyasdirin** [googlecodelabs/android-topeka#4]
Error:Unable to find method 'com.android.build.gradle.tasks.factory.AndroidUnitTest.setTestClassesDirs(Lorg/gradle/api/file/FileCollection;)V'.
Possible causes for this unexpected error include:<ul><li>Gradle's dependency cache may be corrupt (this sometimes occurs after a network connection timeout.)
<a href="syncProject">Re-download dependencies and sync project (requires network)</a></li><li>The state of a Gradle build process (daemon) may be corrupt. Stopping all Gradle daemons may solve this problem.
<a href="stopGradleDaemons">Stop Gradle build processes (requires restart)</a></li><li>Your project may be using a third-party plugin which is not compatible with the other plugins in the project or the version of Gradle requested by the project.</li></ul>In the case of corrupt Gradle processes, you can also try closing the IDE and then killing all Java processes.

**Comment by gen0083** [comment]
I faced same issue.
You can avoid this to update gradle wrapper version.
See my pr #5

**Comment by chenzhuo914** [comment]
Just upgrade the gradle version in the project's build.gradle file:

```
- classpath 'com.android.tools.build:gradle:3.0.0-alpha3'
+ classpath 'com.android.tools.build:gradle:3.0.0-alpha6'
```

**Comment by jkingyens** [comment]
Thanks for reporting this. Should be fixed with fc347d8e98eed835ea2398a7a4531f2f0055a4cf
